tests/console/yast2_samba.pm

```diff
@@ -62,7 +62,7 @@ sub run {
     assert_screen 'yast2_console-finished';
 
     # check ldap server status at first, a local ldap server is needed in the test case
-    assert_script_run "systemctl show -p ActiveState slapd.service | grep ActiveState=active";
+    systemctl 'show -p ActiveState slapd.service | grep ActiveState=active';
```

Comment:
As above, not only action on service. Maybe worth adding functionality to systemctl subroutine.